Puppetfile

```diff
@@ -7,12 +7,12 @@ mod 'camptocamp/augeas', '1.1.0'
 mod 'elasticsearch/elasticsearch'
 mod 'fsalum/redis'
 mod 'maestrodev/wget', '>= 1.1.0'
-mod 'puppetlabs/apt'
```

Comment:
This is similar to the conversation on #3971 but I understand it much better than I did 3 weeks ago :). I think we should get better at specifying exact versions to prevent unintended changes.